Components/Image/image.html

```diff
@@ -0,0 +1,244 @@
+                        <a
+                            href="/Components/Button/button.html"
+                            class="drawer-link"
+                            >Button</a
+                        >
+                        <a href="/Components/Card/card.html" class="drawer-link"
+                            >Card</a
+                        >
+                        <a
+                            href="/Components/Image/image.html"
+                            class="drawer-link"
+                            >Image</a
+                        >
+                        <a
+                            href="/Components/Input/input.html"
+                            class="drawer-link"
+                            >Input</a
+                        >
+                    </div>
+                </aside>
+            </div>
+
+            <div>
+                <div class="component-content-container">
+                    <div class="component-nav-container">
+                        <a
+                            href="/Components/Card/card.html"
+                            class="component-nav-link"
+                        >
+                            <i class="material-icons component-nav-arrow">
+                                arrow_left
+                            </i>
+                            <span>Card</span>
+                        </a>
+                        <a
+                            href="/Components/Input/input.html"
+                            class="component-nav-link"
+                        >
+                            <span>Input</span
+                            ><i class="material-icons component-nav-arrow">
+                                arrow_right
+                            </i>
+                        </a>
+                    </div>
+
+                    <h1 class="component-heading">Image</h1>
+
+                    <h3 class="component-subheading">
+                        Image component CSS - import url
+                    </h3>
+                    <iframe
+                        src="https://www.thiscodeworks.com/embed/61fd86e033c09d0015b44df6"
+                        style="
+                            width: 54rem;
+                            height: 6.875rem;
+                            margin: 1em 0;
+                            max-width: 100%;
+                        "
+                        frameborder="0"
+                    ></iframe>
+
+                    <h3 class="component-subheading">
+                        Image component CSS - HTML link tag
+                    </h3>
+                    <iframe
+                        src="https://www.thiscodeworks.com/embed/61fd871033c09d0015b44df7"
+                        style="
+                            width: 54rem;
+                            height: 6.875rem;
+                            margin: 1em 0;
+                            max-width: 100%;
+                        "
```

Comment:
Use it in css file

Reply:
Idk why for iframes class is not working, so issue carbon also had the same issue. I tried but didn't work, so inline styles for iframes only nowhere else.